css/component.css

```diff
@@ -0,0 +1,74 @@
+.side-content{
+    font-family: 'Roboto', sans-serif;
+    width: 100%;
+    margin: 50px auto 0;
```

Comment:
As suggested above, rem units can be used.

Reply:
change has been done